prov/efa/src/efa_hmem.c

```diff
@@ -56,6 +58,12 @@ static int efa_domain_hmem_info_init_protocol_thresholds(struct efa_domain *efa_
 		fi_param_get_size_t(&efa_prov, "runt_size", &info->runt_size);
 		fi_param_get_size_t(&efa_prov, "inter_min_read_message_size", &info->min_read_msg_size);
 		fi_param_get_size_t(&efa_prov, "inter_min_read_write_size", &info->min_read_write_size);
+		if (-FI_ENODATA != fi_param_get(&efa_prov, "inter_max_medium_message_size", &tmp_value)) {
```

Comment:
Just a tip (no need to change this here): you can avoid declaring a temp value for throwaway vars like these by passing a compound literal as an "anonymous" value.

```suggestion
		if (-FI_ENODATA != fi_param_get(&efa_prov, "inter_max_medium_message_size", &(size_t){0})) {
```